src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,129 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
```

Comment:
I notice that you seem missing the heading of the class such as 
/**
    Storage class to ....
**/